packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -352,6 +352,20 @@ skipInNode(describe)('plugin-meetings', () => {
           assert.equal(alice.meeting.audio.muted, false);
         }));
 
+      it('alice mute bob, bob unmute self ', () => alice.meeting.mute(bob.meeting.members.selfId, true)
```

Comment:
Just to make it plainly clear
```suggestion
      it('alice mute bob audio, bob unmute self ', () => alice.meeting.mute(bob.meeting.members.selfId, true)
```